app/controllers/global_statistics_controller.rb

```diff
@@ -0,0 +1,5 @@
+class GlobalStatisticsController < ApplicationController
+  def show
```

Comment:
can we rename this action to something like `download` ?

Reply:
changed